src/Components/test/E2ETest/Tests/FormsTest.cs

```diff
@@ -74,6 +74,26 @@ public async Task EditFormWorksWithDataAnnotationsValidator()
         Browser.Equal("OnValidSubmit", () => appElement.FindElement(By.Id("last-callback")).Text);
     }
 
+    [Fact]
+    public void EditFormWorksWithDataAnnotationsValidatorAndDI()
+    {
+        var appElement = Browser.MountTestComponent<ValidationComponentDI>();
+        var form = appElement.FindElement(By.TagName("form"));
+        var userNameInput = appElement.FindElement(By.ClassName("the-quiz")).FindElement(By.TagName("input"));
+        var submitButton = appElement.FindElement(By.CssSelector("button[type=submit]"));
+        var messagesAccessor = CreateValidationMessagesAccessor(appElement);
+
+        userNameInput.SendKeys("Jaws\t");
+        submitButton.Click();
+        //We can only have this errormessage when DI is working
+        Browser.Equal(new[] { "Steven Spielberg did not star in this movie" }, messagesAccessor);
```

Comment:
😄 can we actually use some generic text for this test (test class and component). As it stands we _may_ have to do some attribution. @guardrex may have [some more context](https://github.com/dotnet/AspNetCore.Docs/pull/22718#issuecomment-879727520) (or if that only applies to documentation).

Reply:
*Indeed!* CELA is pleased to see attribution and **_zero lawsuits_** hitting the company. 😆 lol

Reply:
😀 Hope this is more appropriate.

Reply:
fyi: https://youtu.be/WcnuI8o64Io?t=95 You can see Steven Spielberg acting in Gremlins 😋